apps/editing-toolkit/editing-toolkit-plugin/help-center/src/admin-bar.js

```diff
@@ -0,0 +1,56 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useHappychatAvailable } from '@automattic/happychat-connection';
+import HelpCenter from '@automattic/help-center';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
+import * as ReactDOM from 'react-dom';
+import { QueryClientProvider } from 'react-query';
+import { whatsNewQueryClient } from '../../common/what-new-query-client';
+import CalypsoStateProvider from './CalypsoStateProvider';
+
+function AdminHelpCenterContent() {
+	const { setShowHelpCenter } = useDispatch( 'automattic/help-center' );
+	const show = useSelect( ( select ) => select( 'automattic/help-center' ).isHelpCenterShown() );
+	const { data } = useHappychatAvailable();
+	const button = document.getElementById( 'wp-admin-bar-help-center' );
+
+	useEffect( () => {
+		if ( data?.status === 'assigned' ) {
+			setShowHelpCenter( true );
+		}
```

Comment:
The Help Center manages his own showing/hiding and checking for sessions. You probably don't need this.